cmds/core/io/smn.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 the u-root Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build amd64
+
+package main
+
+import (
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+type op func() error
+
+// This is a const at present but there are no guarantees.
+const pcibase = 0xe0000000
+
+func init() {
+	usageMsg += `io rs index # read from system management network
```

Comment:
Is this an AMD specific thing? Can you add some docs if so?

Reply:
added